src/components/navigation/ProfileDropdown.tsx

```diff
@@ -80,6 +80,7 @@ const ProfileDropdown = () => {
           </Query>
           <DropdownList
             selectedIndex={-1}
+            width={130}
```

Comment:
May not need this if we keep `fit-content` and set minWidth instead for the other dropdowns?